AMP-87690 Add IP range for CDP related help doc (#47)
amplitude/amplitude-dev-center: docs/data/destination-cohort-overview.md

- Daily sync cohorts have a maximum of 10 million users.
- If you archive an existing cohort sync, the next sync will fail and the scheduled sync will be disabled.
- When you archive a cohort sync, the next sync fails and Amplitude disables any future scheduled syncs.

> - If you need to whitelist or configure any network settings, make sure to allow traffic from this specific IP address to ensure seamless. The IP addresses based on whether you're using EU or US Amplitude data centers.

Reviewer: ```suggestion
- Allow traffic from the following IP addresses, based on your region:
```
